lib/web/app/auth.go

```diff
@@ -69,7 +69,7 @@ func (h *Handler) startAppAuthExchange(w http.ResponseWriter, r *http.Request, p
 	// if required apps is length 1, it _should_ be itself as the final stop of the redirect chain.
 	// If that is the case, skip any further required app redirects and complete the auth flow.
 	// This value gets shifted off the front after a complete auth exchange
-	if len(requiredApps) > 1 && requiredApps[0] != reqAddr.Host() {
+	if len(requiredApps) > 1 && requiredApps[0] != reqAddr.String() {
```

Comment:
wont this break existing required app config? If we previously only checked for host with `reqAddr.Host()` but now `reqAddr.String()` returns `host:port`?

Reply:
Nah, existing requiredApp config is app name only already, no fqdn/url